tests/epyccel/test_array_as_func_args.py

```diff
@@ -135,3 +135,14 @@ def array_complex_2d_scalar_add(x : 'T[:,:]', a : 'T', d1 : int, d2 : int):
         f2(x2, a, d1, d2)
 
         assert np.array_equal( x1, x2 )
+
+def test_array_final(language):
+    def array_final(x : 'const float[:]'):
+        return x[0]
```

Comment:
Why using the word final?

Reply:
Final (from typing) is Python's equivalent of const